### PR TITLE
Add NAS-WV03B2 to neo,ts

### PR DESCRIPTION
**Pull request by baggiowu**
I have add NAS-WV03B2 to neo.ts. thanks.

### CODE REVIEW

File: src/devices/neo.ts
Comment (Koenkk) [inline src/devices/neo.ts:1]: Could you change back the file permissions?

File: src/devices/neo.ts [hunk 1]
```diff
@@ -225,6 +225,44 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_a9ojznj8', '_TZE284_a9ojznj8']),
+        model: 'NAS-WV03B2',
+        vendor: 'NEO',
+        description: 'Smart Sprinkler Timer',
```
Comment (Koenkk) [inline src/devices/neo.ts:232]:
```suggestion
        description: 'Smart sprinkler timer',
```

File: src/devices/neo.ts [hunk 2]
```diff
@@ -225,6 +225,44 @@ const definitions: Definition[] = [
...
+            // All datapoints go in here
+            tuyaDatapoints: [
+                [
+                    3,
+                    'status',
+                    tuya.valueConverterBasic.lookup({
+                        Off: tuya.enum(0),
+                        Auto: tuya.enum(1),
+                        Disabled: tuya.enum(2),
+                        'APP manual': tuya.enum(3),
```
Comment (Koenkk) [inline src/devices/neo.ts:255]:
```suggestion
                        'app_manual': tuya.enum(3),
```
Please also make all other entries lowercase and replace spaces with `_`

**Comment by Koenkk** [comment]
Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `MODEL.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

File: src/devices/neo.ts
Comment (baggiowu) [reply src/devices/neo.ts:1]: I have changed "neo.ts" permissions.

File: src/devices/neo.ts (hunk 1 above)
Comment (baggiowu) [reply src/devices/neo.ts]: I have changed many words to lowercase.

File: src/devices/neo.ts (hunk 2 above)
Comment (baggiowu) [reply src/devices/neo.ts]: I have done.

**Comment by baggiowu** [comment]
> Could you also submit a picture for the docs? This can be done by:
> 
> 1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
> 2. Go to the `public/images/devices` directory, _Add file_ -> _Upload files_
> 3. Upload the files and press _Commit changes_
> 4. Press _Contribute_ -> _Open pull request_ -> update title/description -> _Create pull request_
> 
> Make sure that:
> 
> * The filename is `MODEL.png`
> * The size is 512x512
> * The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

yes. I have submit a pull request to zigbee2mqtt.io.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**